src/iris_api/sender/cache.py

```diff
@@ -34,20 +38,45 @@
 target_reprioritization = None
 target_names = None
 targets_for_role = None
+application_quotas = None
+
+
+class IrisAuth(requests.auth.AuthBase):
```

Comment:
Since we are the active sender we don't need to call Iris to send a notification or make an incident. Just assume the Iris identity and send the notifications directly. There might be some code dupe/split but it's better than having increased runtime complexity.